Source/Core/Core/HW/GCPadEmu.cpp

```diff
@@ -50,7 +50,10 @@ GCPad::GCPad(const unsigned int index) : m_index(index)
   // buttons
   groups.emplace_back(m_buttons = new ControllerEmu::Buttons(_trans("Buttons")));
   for (unsigned int i = 0; i < sizeof(named_buttons) / sizeof(*named_buttons); ++i)
-    m_buttons->controls.emplace_back(new ControllerEmu::Input(named_buttons[i]));
+  {
+    const std::string& ui_name = (named_buttons[i] == "Start") ? "START" : named_buttons[i];
```

Comment:
What kind of comment? I guess I could say something like "We use a different `ui_name` than `name` for Start because the name we want to display in the UI isn't the same as the name we're using in the INI", but that's already implied by using a different `ui_name` than `name`, as there's no other reason to have the two differ...